components/TextFields/src/ContainedInputView/MDCFilledTextField.m

```diff
@@ -24,6 +24,7 @@ @interface MDCFilledTextField ()
 @end
 
 @implementation MDCFilledTextField
+@dynamic borderStyle;
```

Comment:
nit: I think if we mark just the base class's API as NS_UNAVAILABLE and not re declare them NS_UNAVAILABLE in the subclasses we wont have to @dynamic ly define them here.

Reply:
I'd prefer to leave it an available API on the base class though. The gif in the PR description shows a `.roundedRect` border style in action on an MDCBaseTextField (the bottom textfield)  and I think it looks good :/